templates/beta-post.adoc

```diff
@@ -0,0 +1,186 @@
+
+// // // // // // // //
+// Copy feature from both lists in the previous beta post creating    // a cumulative list from all previous betas to make a list of all    // existing features in the new beta.
+// // // // // // // //
+
+=== Try it now
+
+To try out these Jakarta EE 9 features on Open Liberty in a lightweight package, just update your build tools to pull the Open Liberty Jakarta EE 9 Beta Features package instead of the main release. The beta works with Java SE 14, Java SE 11, or Java SE 8.
+
+If you're using link:{url-prefix}/guides/maven-intro.html[Maven], here are the coordinates:
+
+[source,xml]
+----
+<dependency>
+    <groupId>io.openliberty.beta</groupId>
+    <artifactId>openliberty-jakartaee9</artifactId>
+    <version>20.0.0.RELEASE_VERSION-beta</version>
+    <type>zip</type>
+</dependency>
+----
+
+Or for link:{url-prefix}/guides/gradle-intro.html[Gradle]:
+
+[source,gradle]
+----
+dependencies {
+    libertyRuntime group: 'io.openliberty.beta', name: 'openliberty-jakartaee9', version: '[20.0.0.RELEASE_VERSION-beta,)'
+}
+----
+// // // // // // // //
+// Above:
+// Replace RELEASE_VERSION with the version number of Open Liberty
+// // // // // // // //
+Or take a look at our link:{url-prefix}/downloads/#runtime_betas[Downloads page].
+
+Enable the Jakarta EE 9 beta features in your app's `server.xml`. You can enable the individual features you want (but remember if you enable the `jsp-3.0` feature, the `servlet-5.0` and `el-4.0` features are automatically enabled for you) or you can just add the Jakarta EE 9 convenience feature to enable all of the Jakarta EE 9 beta features at once:
+
+[source, xml]
+----
+  <featureManager>
+    <feature>jakartaee-9.0</feature>
+  </featureManager>
+----
+
+Or you can add the Web Profile convenience feature to enable all of the Jakarta EE 9 Web Profile beta features at once:
+
+[source, xml]
+----
+  <featureManager>
+    <feature>webProfile-9.0</feature>
+  </featureManager>
+----
+
+=== HEADING
+
+// // // // // // // //
+// Remove if nothing else to say, or provide an update on aspect of the Jakarta EE 9 beta e.g- TCK
+// // // // // // // //
+
+[#allbeta]
+== All Beta Features package
+
+As well as the Jakarta EE 9 features listed above, the new All Beta Features package contains our new FEATURES features which have been available in our link:/downloads/#development_builds[nightly builds] for a few weeks but are now easy to install from Maven Central too.
+// // // // // // // //
+// Above:
+// Change FEATURES to a feature that is included in this release     // version.
+// // // // // // // //
+
+[#SUB_TAG_1]
+=== SUB_FEATURE_TITLE
+
+// // // // // // // //
+// FURTHER EXPLENATION OF THE FEATURE/CHANGE ABOVE
+// // // // // // // //
+
+
+
+[source, java]
+----
+// // // // // // // //
+// EXAMPLE CODE
+// // // // // // // //
+----
+
+
+=== Try it now section for the feature 
```

Comment:
Ah, this is not "for the feature". This is just how to try the All Beta Features package. This section will appear just once at the end of the All Beta Features section, like there is for the Jakarta EE 9 package section. So two "Try it now" sections in the whole blog post.

However, each feature in the All Beta Features section will probably also have its own server.xml example showing what feature name to add to the server.xml.

Reply:
Done